examples/wallet/send_basic_output.c

```diff
@@ -9,22 +9,23 @@
 #include <inttypes.h>
 #include <stdio.h>
 
+#include "wallet/output_basic.h"
 #include "wallet/wallet.h"
 
 #define Mi 1000000
 
 #define NODE_HOST "localhost"
-#define NODE_PORT 443
-#define NODE_USE_TLS true
-#define TEST_COIN_TYPE SLIP44_COIN_TYPE_SHIMMER
+#define NODE_PORT 14265
+#define NODE_USE_TLS false
+#define TEST_COIN_TYPE SLIP44_COIN_TYPE_IOTA
 
 // replace this with your mnemonic string
 static char const* const test_mnemonic =
     "acoustic trophy damage hint search taste love bicycle foster cradle brown govern endless depend situate athlete "
     "pudding blame question genius transfer van random vast";
 static char const* const bech32_receiver =
-    "rms1qrhacyfwlcnzkvzteumekfkrrwks98mpdm37cj4xx3drvmjvnep6xrlkcfw";  // receiver address in bech32 format
-uint32_t const sender_addr_index = 0;                                   // address index of the wallet
+    "atoi1qpswqe4v8z2cdtgc7sfj0hfneqh37lhmjgnth36mfndwcxkjrakcvx42lsn";  // receiver address in bech32 format
```

Comment:
it's not related to this PR but could you change this hard-coded bech32 address to the 2nd address index? so this example can work with different network configurations.